### PR TITLE
Add sensor platform to Mealie

### PR DESCRIPTION
**Pull request by joostlek**
## Proposed change

Add sensor platform to Mealie

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/122280
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced support for sensors in the Mealie integration, allowing users to access statistics on recipes, categories, tags, tools, and users.
	- Enhanced operational capabilities of the Mealie integration for improved tracking and management of meal planning resources.
- **Documentation**
	- Updated integration metadata to reflect the new sensor feature and added a dedicated section for sensor statistics.

### CODE REVIEW

File: source/_integrations/mealie.markdown [hunk 1]
```diff
@@ -46,6 +47,10 @@ The integration will create a calendar for every type of meal plan:
 
 The integration will create a to-do list for every Mealie shopping list.
 
+## Sensors
+
+The integration provides sensors for the statistics like the number of recipes, categories, tags, tools and users.
```
Comment (c0ffeeca7) [inline source/_integrations/mealie.markdown]: I think it could be helpful to have a list of all the sensor names with a description.

Comment (joostlek) [reply source/_integrations/mealie.markdown]: It actually are those 5 haha

Comment (c0ffeeca7) [reply source/_integrations/mealie.markdown]: Ah ok. Can you be a bit more descriptive? What are the categories, what are tags and tools?

Comment (joostlek) [reply source/_integrations/mealie.markdown]: They are taxonomies within Mealie, so you have tools added to cook with, same with categories recipes are in.

They are also placed on the homepage of Mealie so I am assuming they are quite a well known name for the users.

cc @andrew-codechimp

Comment (andrew-codechimp) [reply source/_integrations/mealie.markdown]: These taxonomies are deliberately vague within Mealie to allow people to adapt them to their needs.
As an example I use Categories for meal types and also things like baking, seasonings, etc. whilst Tags I use to indicate vegetarian and cuisine type, quick meals etc.
These should be considered a bit like Labels & Categories within HA, there's no one way to describe them, anyone using Mealie will know what they are.

Comment (c0ffeeca7) [inline source/_integrations/mealie.markdown]:
```suggestion
The integration provides the following sensors for the statistics:
- number of recipes
- categories (such as beverage, dessert, Italian, seafood)
- tags (such as alcohol)
- tools (such as instant pot, air fryer, or BBQ)
- users
```
Would it hurt to give an example?

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]
couldn't find an example for tools

**Comment by andrew-codechimp** [comment]
> couldn't find an example for tools

Instant Pot, Air Fryer, BBQ are what I have.

File: source/_integrations/mealie.markdown (hunk 1 above)
Comment (joostlek) [reply source/_integrations/mealie.markdown]: Oh that's nice, haven't thought about describing it like that

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @joostlek :+1:

**Comment by joostlek** [comment]
Oh thanks for adding the examples

**Pull request merged**